distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -1,3 +1,230 @@
+Location:: an example of a geographically constrained search.
+Temporal:: an example of a temporally constrained search.
+
+==== Configuring a Workspace in ${catalog-ui}
+
+Configure each workspace with standing queries and share options.
+
+.Adding searches
+. Select the default name of the workspace (*New Workspace* or template name) to change to a unique name.
+. If *Blank* workspace was selected, select *Add Search* to add the first search. Each workspace can have up to ten searches.
+.. Select *Basic* Search to select simple search criteria, such as *text*, *time*, and *location*.
+.. Select *Advanced* Search to access a query-builder for more complex queries.
+
+.Navigator Menu Options
+* *Workspaces*: View all available workspaces.
+* *Upload*: Interface to add resources to catalog.
+* *Open Workspaces* (if applicable): Shows a list of open workspaces.
+
+.Workspace Menu Options
+* Select the *Menu* button next to the Workspace name.
+** *Save*: Save changes to the workspace.
+** *Run All Searches*: Start all saved searches within this workspace.
+** *Cancel All Searches*: Cancel all running searches.
+** *Open in New Tab*: Opens this workspace in a separate tab.
+** *View Sharing*: View and edit settings for sharing this workspace. (User must be signed in to share or view shared workspaces.)
+** *View Details*: View the current details for a cloud-based workspace (User must be signed in).
+** *Duplicate*: Create a copy of this workspace.
+** *Subscribe*: Receive updates on this workspace.
+** *Move to Trash*: Delete (archive) this workspace.
+
+==== Sharing Workspaces
+
+In the ${catalog-ui}, workspaces can be shared between users at different levels of access as needed.
+
+.Share a Workspace
+. Select the Workspace.
+. Select the *Options* menu.
+. Select *View Sharing*.
+.. To share by user role, set the drop-down menu to *Can Access* for each desired role. All users with that role will be able to view.
+.. To share with an individual user, add their email to the email list.
+. Click *Apply*.
+
+.Remove Sharing on a Workspace
+. Select the Workspace.
+. Select the *Options* Menu.
+. Select *View Sharing*.
+.. To remove the workspace from users with specific roles, set the drop-down menu to *No Access* for that role.
+.. To remove individual users, remove their email from the email list.
+
+=== Searching with ${catalog-ui}
+
+The Search pane has three tabs: *Searches*, *Results*, and *Bookmarks*.
+
+==== Searches Tab
+
+View, edit, and add searches from this tab.
+
+.Search Menu Options
+* *Run*: Trigger this search to begin immediately.
+* *Stop*: Stop this search.
+* *Delete*: Remove this search
+* *Duplicate*: Create a copy of this search as a starting point
+* *Search Archived*: Execute the search, but specifically for archived results.
+
+===== Editing a Standing Search
+
+Existing searches and their settings can be updated by clicking on the search in the *Searches* tab of the workspace.
+
+* *Basic*
```

Comment:
Agreed. For example, there are special characters that have special meanings in the `Text` section that would be helpful to call out, discussion about how to draw shapes for location searches and what coordinate systems are supported, what the types in `Match Types` are used for, etc. This section could be very extensive. The search capabilities are the heart of the Catalog UI. 

Need the following new subsections to be parallel with the other sub-sections.
- `Editing a Basic Search`
- `Editing an Advanced Search`